src/scripts/api/types/books.ts

```diff
@@ -0,0 +1,64 @@
+import { Links, Meta, Predicate } from './general';
+
+export interface ABook {
```

Comment:
I didn't saw naming convention to name interfaces with starting `A`, but that's really up to you. The most common that I saw - starting with `I` or ending with `Props`